docassemble/MAVirtualCourt/data/questions/basic-questions.yml

```diff
@@ -329,14 +339,16 @@ buttons:
 ---
 id: other users there are any
 question: |
-  Is there anyone on the other side of this case?
+  Is there any person or organization on the other side of this case?
 subquestion: |
-  I.e., the person who you are suing or who is suing you.
+  % if users == plaintiffs:
+  Answer yes if there is a person or organization you are suing or taking to court.
+  % endif
 yesno: other_parties.there_are_any
 ---
 id: children names without birthdate
 question: |
-  Tell us about the children who will be on this form
+  Tell us about the children involved in this case.
```

Comment:
Still reviewing, no errors, just pointing out that this language is different than the other children question language.